src/ngraph_encapsulate_op.cc

```diff
@@ -443,6 +437,37 @@ class NGraphEncapsulateOp : public OpKernel {
       }
       ng_exec = it->second;
     }
+    return Status::OK();
+  }
+
+  //---------------------------------------------------------------------------
+  // OpKernel::Compute
+  //---------------------------------------------------------------------------
+  void Compute(OpKernelContext* ctx) override {
+    std::ostringstream oss;
+    oss << "Execute: Encapsulate_" << my_instance_id << ": " << name();
+    ngraph::Event event(oss.str(), name(), "");
+
+    Timer compute_time;
+    std::lock_guard<std::mutex> lock(m_compute_lock);
+    NGRAPH_VLOG(4) << "NGraphEncapsulateOp::Compute starting for cluster "
+                   << m_ngraph_cluster;
+
+    NGRAPH_VLOG(4) << "Compute: Got backend of type: " << m_op_backend_name;
+    ng::runtime::Backend* op_backend =
+        BackendManager::GetBackend(m_op_backend_name);
+
+    ngraph::Event event_func_maybe_create("FunctionMaybeCreate", name(), "");
+    Timer function_lookup_or_create;
+
+    std::vector<TensorShape> input_shapes;
+    std::vector<const Tensor*> static_input_map;
+    std::shared_ptr<ngraph::Function> ng_function;
+    std::shared_ptr<ngraph::runtime::Executable> ng_exec;
+
+    // Get ngraph executable and inputs information
+    OP_REQUIRES_OK(ctx,
```

Comment:
Add log for this function call

Reply:
I am using `OP_REQUIRES_OK` and not `OP_REQUIRES`, so it will already print logs from the function. Is that enough?
Or should i do try catch and add logs if this function returns bad status?

Reply:
I meant just a log like "Getting executable " in lines of this,
https://github.com/NervanaSystems/ngraph-tf/blob/master/src/ngraph_encapsulate_op.cc#L467

To signal where in the compute function we are.